src/Scripting/CoreTest/TestCompilationFactory.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.VisualBasic;
```

Comment:
I think it's in the Core as it is shared by both CSharpScriptingTests (portable) and CSharpScriptingTests.Desktop. Should this class be moved to the portable version and then a reference added for desktop tests?